mne/viz/_3d.py

```diff
@@ -267,10 +267,11 @@ def _plot_mri_contours(mri_fname, surf_fnames, orientation='coronal',
 
 @verbose
 def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
-               ch_type=None, source=('bem', 'head'), coord_frame='head',
-               meg_sensors=('helmet', 'sensors'), eeg_sensors='original',
-               dig=False, ref_meg=False, ecog_sensors=True, head=None,
-               brain=None, verbose=None):
+               ch_type=None, source=('bem', 'head', 'outer_skin',
+                                     'outer_skull', 'inner_skull'),
```

Comment:
This signature is a bit too long. To make things simpler to parse I'd make this `source='auto'` and say that `'auto'` is an alias